source/_integrations/husqvarna_automower.markdown

```diff
@@ -118,6 +120,28 @@ The integration will create a calendar entity for all mowers. The calendar shows
 
 The integration will create a device tracker entity to show the position of the mower.
 
+### Event (if available)
+
+- Shows the last error as event.
+- Includes additional context: `severity`, `latitude`, `longitude`, and `date_time`.
+
+#### Example attributes
+
+| Attribute     | Description                            |
+|---------------|----------------------------------------|
+| `event_type`  | Error code (for example, `tilt_error`)        |
+| `severity`    | Error severity (for example, `error`, `warning`) |
+| `latitude`    | Latitude where the error occurred      |
+| `longitude`   | Longitude where the error occurred     |
+| `date_time`   | Timestamp of the error                 |
+
+#### Use cases
+
+- Send a notification when the mower is lifted or stuck.
+- Show last error location on a map
+
+ℹ️ **Info:** The entity will only be created when a new message is received. If a mower hasn’t reported any errors yet, the entity won't show up.
```

Comment:
do you want to wrap this in a {% note %}{% endnote %}, or {% tip %}{% endtip %}?